demo/components/tree-view-demo.tsx

```diff
@@ -64,6 +114,19 @@ export class TreeViewDemo extends React.Component<{}, TreeViewDemoState> {
                         selectedItem={this.state.selectedItem}
                     />
                 </section>
+                <h3>TreeView with custom item renderer</h3>
+                <section>
+                    <SelectedItem selectedItem={this.state.selectedItemCustom} />
+                    <br />
+                    <TreeView
```

Comment:
Please separate your demo to two components

Reply:
why separate? this component thing represents the demo for the <TreeView /> as a whole.
will test it as you suggested